opm/simulators/flow/Main.hpp

```diff
@@ -433,10 +433,6 @@ namespace Opm
 
                 Opm::FlowMainEbos<PreTypeTag>::printPRTHeader(outputCout_);
 
-                if (outputCout_) {
-                    OpmLog::info("Reading deck file '" + deckFilename + "'");
```

Comment:
I now agree with removing this printout from this spot, but would like it to be added at readDeck.cpp line 192 (before the checkKeywords() call).